### PR TITLE
Revert "[apm,python] Add httpx compatibility, and fix missing and misnumbered links"

### PR DESCRIPTION
**Pull request by brettlangdon**
Reverts DataDog/documentation#11243

### CODE REVIEW

**Comment by brettlangdon** [comment]
This wasn't meant to be merged yet, sorry!

**Review by jtappa — APPROVED** [review]

**Comment by jtappa** [comment]
Woops! Please open a new PR but add the "Do Not Merge" label until it's ready for review and merge on our end, thanks!

**Pull request merged**
